sensu-plugins-process-checks.gemspec

```diff
@@ -31,11 +31,11 @@ Gem::Specification.new do |s| # rubocop: disable Metrics/BlockLength
   s.test_files             = s.files.grep(%r{^(test|spec|features)/})
   s.version                = SensuPluginsProcessChecks::Version::VER_STRING
 
-  s.add_runtime_dependency 'english', '0.6.3'
+  s.add_runtime_dependency 'english', '0.7.0'
```

Comment:
Are you sure you want to pin to an exact version again?

Reply:
I imagine '~> 0.7' would be safe.